packages/cli/src/executions/execution.types.ts

```diff
@@ -37,12 +39,38 @@ export declare namespace ExecutionRequest {
 	type Retry = AuthenticatedRequest<RouteParams.ExecutionId, {}, { loadWorkflow: boolean }, {}>;
 
 	type Stop = AuthenticatedRequest<RouteParams.ExecutionId>;
-
-	type GetManyActive = AuthenticatedRequest<{}, {}, {}, { filter?: string }>;
 }
 
-export type GetManyActiveFilter = {
-	workflowId?: string;
-	status?: ExecutionStatus;
-	finished?: boolean;
-};
+export namespace ExecutionSummaries {
```

Comment:
I guess this means we're changing the contract heavily right? Just making sure that Alex is onboard with it.

Reply:
This type is the result of the parsing done by the middleware. I tested with the UI and it works. From what I can tell this does not change the contract.